teuthology/orchestra/remote.py

```diff
@@ -638,8 +634,7 @@ class VirtualConsole():
     """
     Virtual Console (set from getRemoteConsole)
     """
-    def __init__(self, name, ipmiuser, ipmipass, ipmidomain, logfile=None,
```

Comment:
oh...because we have a factory function, we can just elide the unused param.  I see.